Doc/whatsnew/3.13.rst

```diff
@@ -122,11 +122,21 @@ Optimizations
 Deprecated
 ==========
 
+Pending Removal in Python 3.15
+------------------------------
+
 * :mod:`wave`: Deprecate the ``getmark()``, ``setmark()`` and ``getmarkers()``
   methods of the :class:`wave.Wave_read` and :class:`wave.Wave_write` classes.
   They will be removed in Python 3.15.
   (Contributed by Victor Stinner in :gh:`105096`.)
 
+Pending Removal in Python 3.16
+------------------------------
+
+* :mod:`array`'s ``'u'`` format code, deprecated in docs since Python 3.3,
+  emits :exc:`DeprecationWarning` since 3.13
+  and will be removed in Python 3.16.
```

Comment:
Can you suggest a replacement? I suppose that the ``'w'`` format should be used instead.